### PR TITLE
Wire up event calendar page

### PR DESCRIPTION
**Pull request by jtorreggiani**
## Ticket Description
This PR implements changes to address https://github.com/data-umbrella/event-board-web/issues/13.

## Description of Changes
Wires up the event search form to work with the event calendar. Also, adds the event list to the bottom. The display and user experience of the calendar using the plugin we have is rough. We will need to discuss what functionality is needed for mobile calendar. For now, I am just rendering the event list grouped by month (which also need mobile design polish).

## Before and After for UI Updates

Before:

Desktop 
Screen Shot 2022-09-10 at 10 58 58 PM

Mobile
Screen Shot 2022-09-10 at 11 01 16 PM

After:

Desktop

Screen Shot 2022-09-10 at 10 56 37 PM

Screen Shot 2022-09-10 at 10 56 48 PM

Screen Shot 2022-09-10 at 10 56 55 PM

Mobile
Screen Shot 2022-09-10 at 10 59 58 PM

Screen Shot 2022-09-10 at 11 00 13 PM

## For PR Reviewer
- [ ] Does this file change the yarn.lock, package.json or package-lock.json file? If so, why?
- [ ] If this pr contains mobile and desktop changes, did you test on IphoneXr and desktop views?
- [ ] Does this file match the related tickets linked figma file, or does it pass the visual smell test?
- [ ] If this file contains javascript, does the javascript pass the smell test?
- [ ] If you don't feel super confident in your review, did you assign someone more senior to double check?

### CODE REVIEW

File: src/components/pages/EventCalendarPage.jsx [hunk 1]
```diff
@@ -1,158 +1,26 @@
...
-    <div className="mt-4">
-      <FullCalendar
-        plugins={[ dayGridPlugin ]}
-        initialView="dayGridMonth"
-        events={CALENDAR_EVENTS}
-        eventContent={renderEventContent}
-      />
+    <div className="mt-4 mb-12 md:px-6">
+      <div>
+        <h1 className="text-xl font-bold mb-2 mt-12">Search Events</h1>
```
Comment (dashcraft) [inline src/components/pages/EventCalendarPage.jsx:13]: Comment (non-blocking):
Should be an h2 from what i can tell on the figma

**Review by dashcraft — DISMISSED** [review]
1 comment, non blocking, looks significantly better!

File: src/components/pages/EventCalendarPage.jsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ function EventCalendarPage() {
   return (
     <div className="mt-4 mb-12 md:px-6">
       <div>
-        <h1 className="text-xl font-bold mb-2 mt-12">Search Events</h1>
+        <h2 className="text-2xl font-bold mb-2 mt-12">Search Events</h2>
```
Comment (jtorreggiani) [inline src/components/pages/EventCalendarPage.jsx:13]: Updated text sign to max Figma 24px.

**Comment by jtorreggiani** [comment]
@dashcraft, I think this is good to go now. I added some barebones tests for this but didn't want to sink more time in debugging the timezone stuff in jest and the mocks. I will follow up in another PR for more backfilling of tests. If you think this looks good feel free to approve and merge it in.

**Pull request merged**